docs/specification/1.0.md

```diff
@@ -484,21 +484,9 @@ As the CDS Service is executing on behalf of a user, it is important that the da
 
 ### Cross-Origin Resource Sharing
 
-[Cross-origin resource sharing (CORS)](https://developer.mozilla.org/en-US/docs/Web/HTTP/Access_control_CORS) is web security mechanism that is built into browsers. In short, CORS allows servers to control how browsers access resources on the server, including the accessible HTTP response headers. CORS is only honored by web browsers and as such, is a client-side security mechanism.
+[Cross-origin resource sharing (CORS)](https://www.w3.org/TR/cors/) is a W3C standard mechanism that uses additional HTTP headers to enable a web browser to gain permission to access resources from an Internet domain different from that from which the browser is currently accessing.  CORS is a client-side security mechanism with well-documented security risks.
 
-For CDS Services, implementing CORS is required if your CDS Service is to be called from a web browser. As the [CDS Hooks Sandbox](http://sandbox.cds-hooks.org) is a browser application, you must implement CORS to test your CDS Service in the CDS Hooks Sandbox.
-
-You should carefully consider if and how you support CORS in your CDS Service. A secure implementation guide for CORS is outside of the CDS Hooks specification, as it involves implementation decisions that are made based upon your organization's tolerance for risk and deployment environment.
-
-An insecure, extremely permissive, non-production CORS configuration that allows you to test your CDS Service against our Sandbox returns the following HTTP headers:
-
-Header | Value
------- | -----
-Access-Control-Allow-Credentials | true
-Access-Control-Allow-Headers | Authorization, Content-Type
-Access-Control-Allow-Methods | GET, POST, OPTIONS
-Access-Control-Allow-Origin | *
-Access-Control-Expose-Headers | Origin, Accept, Content-Location, Location, X-Requested-With
+Because CDS Services may return decisions involving multiple domains (e.g., link to SMART app),  browser-based EHRs will require CORS support in order to access these services.  A secure implementation guide for CORS is outside of the scope of this CDS Hooks specification. Organizations planning to implement CDS Hooks with CORS support are referred to the Cross-Origin Resource Sharing section of the [OWASP HTML5 Security Cheat Sheet]( https://www.owasp.org/index.php/HTML5_Security_Cheat_Sheet).  
```

Comment:
The example configuration here was extremely useful, and clearly marked as non-production. Do we really need to exclude it from the documentation?

Reply:
The concern @bakerdb had was that our non-secure example was in the specification. I agree this was valuable to help developers get started, so perhaps we should just move this to our (soon to be released) quick start guide (#113)?

Reply:
@brynrhodes @kpshek stated my concern.  I would not advocate hiding the fact that the Sandbox requires CORS, but I don't think the CDS Hooks specification is the right place to discuss the Sandbox.  Quick Start Guide may be a better place to put this information.